app/controllers/api/v1/accounts/agents_controller.rb

```diff
@@ -9,19 +9,16 @@ def index
     @agents = agents
   end
 
-  def destroy
```

Comment:
cleaned up agents controller slightly while enabling it to accept the additional params `availability` and `auto_offline`